packages/material-ui/src/Drawer/Drawer.test.js

```diff
@@ -148,6 +148,32 @@ describe('<Drawer />', () => {
         </Drawer>
       );
 
+      it('should start closed', () => {
+        const wrapper = mount(drawerElement);
+        assert.strictEqual(wrapper.find('Modal').props().open, false);
+        assert.strictEqual(wrapper.contains('Slide'), false);
+      });
+
+      it('should open and close', () => {
+        const wrapper = mount(drawerElement);
+
+        wrapper.setProps({ open: true });
+        wrapper.update();
+        assert.strictEqual(wrapper.find('Slide').props().in, true);
+
+        wrapper.setProps({ open: false });
+        wrapper.update();
+        assert.strictEqual(wrapper.contains('Slide'), false);
+      });
+    });
+
+    describe('opening and closing with keepMounted Modal', () => {
+      const drawerElement = (
+        <Drawer ModalProps={{ keepMounted: true }}>
```

Comment:
Note that the original test(s) failed, as `keepMounted` is defaulted to `false` - The children are unmounted on closed, so the test was failing on line 191 (previously 165):
```js
assert.strictEqual(wrapper.find('Slide').props().in, false);
```